### ISSUE TITLE
Ability to remove toast notification?

### ISSUE THREAD

**Issue by oitptobbe** [ironmansoftware/universal-dashboard#183]
Say I have a button that contains

```
New-UDButton -Id 'unlockbtn' -Text 'Unlock account' -OnClick {
  $toast = Send-UDToast -Message 'Unlocking account' -Duration 2000
  Unlock-ADAccount -Identity $userinfo.samAccountName
  $userinfo.LockedOut = $false
  Remove-UDToast $toast
  Send-UDToast -Message 'Account is unlocked' -Duration 2000
  Remove-UDElement -Id 'unlockbtn'
}
```
I would like to be able to remove toasts dynamically or provide the user with a visual information so that they know that we are doing something.
A "processing" overlay for the page.
Such as 
New-UDPreloader but for the whole page to indicate that we are working on something.
I've tried doing it with New-UDPreloader but I'm unable to "complete" the preloader / remove/ hide? How would that work.

**Comment by adamdriscoll** [comment]
Would showing a modal work for you? #135 

Here's what they would look like: https://materializecss.com/modals.html

You could put spinners and messages in there. 

It's something I'd like to release in the next version.

**Comment by apetitjean** [comment]
Theses toasts would be ultra coooool too :  https://github.com/dolce/iziToast (see "Provide different toast savors" #77)

Some toasts seems to behave like modals though.

**Comment by adamdriscoll** [comment]
@apetitjean I do really like that toast ;) 

@Phyziik If I add modal to the next version, I'll bump to 1.7 but I might sneak out the 1.6.2 bug fix release around June first and then modal will actually come in the next version. I will see how development goes over the next week or so!

**Comment by Phyziik** [comment]
Great, thanks for that!

**Comment by adamdriscoll** [comment]
Implemented via Hide-UDToast in 2.0
